INSTALL.md

```diff
@@ -3,82 +3,94 @@
 Since you are reading this from the Singularity source code, it will be assumed
 that you are building/compiling.
 
+For full instructions on installation, check out our
+[installation guide](https://www.sylabs.io/guides/3.0/user-guide/installation.html).
+
 ## Install system dependencies
+
 You must first install development and libraries to your host.
 Assuming Ubuntu:
 
-```
+```bash
 $ sudo apt-get update && \
-sudo apt-get install -y build-essential \
-libssl-dev uuid-dev libgpgme11-dev squashfs-tools libseccomp-dev pkg-config
+  sudo apt-get install -y build-essential \
+  libssl-dev uuid-dev libgpgme11-dev squashfs-tools libseccomp-dev pkg-config
 ```
 
 On CentOS/RHEL:
 
+```bash
+$ sudo yum groupinstall -y 'Development Tools' && \
+  sudo yum install -y openssl-devel libuuid-devel libseccomp-devel
 ```
-$ sudo yum groupinstall -y 'Development Tools'
-$ sudo yum install -y openssl-devel libuuid-devel libseccomp-devel
-```
-Skip libseccomp-devel on CentOS/RHEL 6.
+Skip `libseccomp-devel` on CentOS/RHEL 6.
```

Comment:
One question before merging. This line may be a bit confusing (I didn't understand it the first time I read it), it says to:
>Skip `libseccomp-devel` on CentOS/RHEL 6.

But the above commands shows to install it?

Maybe to should say:
```
On CentOS/RHEL 6 or less, you may skip `libseccomp-devel`.
```
Just to be a bit more clear about CentOS/RHEL 6.

Reply:
@GodloveD ^^ PING

Reply:
Yeah, that sounds like a good change @WestleyK.  I was also unaware of this.  Maybe needs to be added to docs as well.